src/Symfony/Component/Form/ChoiceList/View/ChoiceView.php

```diff
@@ -30,10 +30,10 @@ class ChoiceView
     /**
      * Creates a new choice view.
      *
-     * @param mixed  $data  The original choice
-     * @param string $value The view representation of the choice
-     * @param string $label The label displayed to humans
-     * @param array  $attr  Additional attributes for the HTML tag
+     * @param mixed       $data  The original choice
+     * @param string      $value The view representation of the choice
+     * @param string|bool $label The label displayed to humans
```

Comment:
I think only `string|false` is used isn't it? Please also explain what false means here.

Reply:
Done

Reply:
Used "discard" word from the docs: https://symfony.com/doc/current/reference/forms/types/choice.html#choice-label